cache/lru_cache.cc

```diff
@@ -557,6 +595,20 @@ void LRUCacheShard::Erase(const Slice& key, uint32_t hash) {
   }
 }
 
+bool LRUCacheShard::IsReady(Cache::Handle* handle) {
+  LRUHandle* e = reinterpret_cast<LRUHandle*>(handle);
+  bool ready = true;
+  if (e->IsPending()) {
+    assert(secondary_cache_);
+    if (e->sec_handle->IsReady()) {
```

Comment:
Should we check if (e->sec_handle != nulllptr)?

Reply:
We could. If ```IsPending()``` is true, it means we're waiting on the secondary cache and ```e->sec_handle``` should be valid, so it might be good to assert it.